index.bs

```diff
@@ -1049,6 +1074,207 @@ Note: Invocations of {{MLContext/compute()}} will fail if any of the {{MLContext
+            dataType: 'float32', shape: [2, 2], readable: true
+          })
+        ]);
+
+    // 4. Initialize the inputs.
+    context.writeTensor(inputTensorA, new Float32Array(4).fill(1.0));
+    context.writeTensor(inputTensorB, new Float32Array(4).fill(0.8));
+
+    // 5. Execute the graph.
+    const inputs = {
+      'A': inputTensorA,
+      'B': inputTensorB
+    };
+    const outputs = {
+      'C': outputTensorC
+    };
+    context.dispatch(graph, inputs, outputs);
+    
+    // 6. Read back the computed result.
+    const result = await context.readTensor(outputTensorC);
+    console.log('Output value:', new Float32Array(result));  // [1, 1, 1, 1]
+  </pre>
+</details>
+</div>
+
+### {{MLContext/createTensor()}}  ### {#api-mlcontext-createtensor}
+
+Creates an {{MLTensor}} associated with this {{MLContext}}.
+
+<div dfn-for="MLContext/createTensor(descriptor)" dfn-type=argument>
+    **Arguments:**
+      - <dfn>descriptor</dfn>: an {{MLTensorDescriptor}}.
+
+    **Returns:** {{Promise}}<{{MLTensor}}>.
+</div>
+
+<details open algorithm>
+  <summary>
+    The <dfn method for=MLContext>createTensor(|descriptor|)</dfn> method steps are:
+  </summary>
+    1. Let |global| be [=this=]'s [=relevant global object=].
+    1. Let |tensor| be the result of [=creating an MLTensor=] given [=this=], and |descriptor|.
+    1. Let |promise| be [=a new promise=].
+    1. Enqueue the following steps to [=this=].{{MLContext/[[timeline]]}}:
+        1. Create |tensor|.{{MLTensor/[[data]]}} given |descriptor| and initialize all bytes to zeros.
+        1. If that fails, then [=queue an ML task=] with |global| to [=reject=] |promise| with an "{{UnknownError}}" {{DOMException}}, and abort these steps.
+        1. Otherwise, [=queue an ML task=] with |global| to [=resolve=] |promise| with |tensor|.
+    1. Return |promise|.
+</details>
+
+### {{MLContext/readTensor(tensor)}}  ### {#api-mlcontext-readtensor}
+
+Reads back the {{MLTensor/[[data]]}} of an {{MLTensor}} from the {{MLContext}}.{{MLContext/[[timeline]]}} to script.
+
+<div dfn-for="MLContext/readTensor(tensor)" dfn-type=argument>
+    **Arguments:**
+      - <dfn>tensor</dfn>: an {{MLTensor}}. The tensor to be read.
+
+    **Returns:** {{Promise}}<{{ArrayBuffer}}>. A buffer containing the result of the read.
+</div>
+
+<details open algorithm>
+  <summary>
+    The <dfn method for=MLContext>readTensor(|tensor|)</dfn> method steps are:
+  </summary>
+    1. Let |global| be [=this=]'s [=relevant global object=].
+    1. Let |realm| be [=this=]'s [=relevant realm=].
+    1. If |tensor|.{{MLGraph/[[context]]}} is not [=this=], then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. If |tensor|.{{MLTensor/[[isDestroyed]]}} is true, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. If |tensor|.{{MLTensor/[[descriptor]]}}["{{MLTensorDescriptor/readable}}"] is false, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
```

Comment:
Could the `readable` getter be referenced by `{{MLTensor/readable}}`? Such as
```suggestion
    1. If |tensor|'s {{MLTensor/readable}} is false, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
```

Reply:
Algorithm steps generally shouldn't be accessing getters, since script can overwrite the getter. Specs do get a bit sloppy here, but referencing the internal state is preferred.

Reply:
Ack. Fixed in [2c9ddfb](https://github.com/webmachinelearning/webnn/pull/787/commits/2c9ddfb7c0808095cbfcd298c9186d2123c1f9ac)

Reply:
@inexorabletash thanks for the clarification! @a-sully , apologize for my misunderstanding.